tox.ini

```diff
@@ -43,6 +43,7 @@ skip_install = true
 commands = check-manifest
 
 [testenv:flake8]
+ignore_list = "S403,S301"
```

Comment:
it's cheating if you ignore the problems instead of solving them!

Reply:
or at least thinking about why the problems are happening and applying appropriate `noqa` tags as in the case of S403, which is about unsafe usage of pickle iirc

Reply:
Those were warnings for pickle being a security vulnerability, I'm going to do some research and implement an alternative in a later PR.

Reply:
there might not be an alternative to using pickle for storage of models, so this is a valid thing to ignore. However, there's a `.flake8` config file where this would be more appropriate instead of `tox.ini`